db/migrate/20180720121837_add_port_status_to_physical_network_ports.rb

```diff
@@ -0,0 +1,5 @@
+class AddPortStatusToPhysicalNetworkPorts < ActiveRecord::Migration[5.0]
+  def change
+    add_column :physical_network_ports, :port_status, :string
```

Comment:
Is this related to this change or did this just sneak in?

Reply:
Sorry for it, this should be in another PR of mine. I'm removing it from here.